src/component/LoginPage.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {Image, Platform, Pressable, Text, View} from 'react-native';
+
+function LoginPage(): React.JSX.Element {
+  const isIOS = Platform.OS === 'ios' ? true : false;
```

Comment:
```suggestion
  const isIOS = Platform.OS === 'ios'
```
삼항 연산자말고 그냥 조건문으로 표기하는게 더 좋을것 같아

Reply:
아 그러네 바꿔야겠다 나 이거 왜 이걸로 만들었지

Reply:
나도 자주 그래 ㅋㅋㅋㅋㅋ 뭔가 조건 들어가면 삼항 연산자부터 찾는 js 개발자들 ㅋㅋㅋㅋㅋㅋㅋ

Reply:
덕분에 코드 문제점이 하나씩 보인다 ㅋㅋㅋ